controllers/k8ssandra/k8ssandracluster_controller.go

```diff
@@ -250,7 +250,11 @@ func (r *K8ssandraClusterReconciler) reconcile(ctx context.Context, kc *api.K8ss
 		}
 
 		if err = remoteClient.Get(ctx, dcKey, actualDc); err == nil {
-			result, err := r.reconcileStargate(ctx, kc, dcTemplate, actualDc, logger, remoteClient)
+			result, err := r.reconcileStargate(ctx, kc, dcTemplate, actualDc, logger, remoteClient, &stargateAuthSchemaReconciled)
+			if !result.IsZero() || err != nil {
+				return result, err
+			}
+			result, err = r.reconcileReaper(ctx, kc, dcTemplate, actualDc, logger, remoteClient, &reaperBackendSchemaReconciled)
```

Comment:
Provided that the schema initialization is done up front, there is no reason that Stargate and Reaper couldn't be deployed at the same time. Just an observation for now 😄